plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -261,6 +263,26 @@ class ServiceDestroyModal extends React.Component {
     );
   }
 
+  /**
+   * Format serviceId to only the service name
+   * e.g path/to/service/name will only return name
+   *
+   * @param {String} serviceId
+   * @returns {String} formatted serviceId
+   *
+   * @memberof ServiceDestroyModal
+   */
+  formatServiceName(serviceId) {
```

Comment:
So... the name of the function is `formatServiceName` and we're passing in `serviceId`. 
From a function named `format*` I'd expect formatting the input, where this function just extracts a part of the input. So my suggestion would be to rename it `extractServiceName` or `getServiceName`